utilities.h

```diff
@@ -153,11 +169,11 @@ std::string ToLower(std::string const &text);
 std::string ToUpper(std::string const &text);
 
 // replaces polish letters with basic ascii
-void win1250_to_ascii( std::string &Input );
-// TODO: unify with win1250_to_ascii()
+void remove_accents( std::string &Input );
+// TODO: unify with remove_accents()
 std::string Bezogonkow( std::string Input, bool const Underscorestospaces = false );
 
-std::string win1250_to_utf8(const std::string &input);
+std::string to_utf8(const std::string &input);
```

Comment:
remove_accents/to_utf8 doesn't really represent that behavior is dependent on settings, maybe file_to_ascii and file_to_utf8?